### ISSUE TITLE
Time for a new official release?

### ISSUE THREAD

**Issue by johnsonjh** [LADSoft/OrangeC#1054]
There has been a lot of work done since 6.73.1 and quite some time as passed ...

**Comment by chuggafan** [comment]
As far as I am aware, the plan is the next release will be after C++17
support is finalized, possibly with finalized C23 support, I am
assuming I'm correct @LADSoft, right?

**Comment by LADSoft** [comment]
yeah that is the plan.   Finish C++17 support and if @chuggafan is done with #embed finish C23 support as well.
here are the things left to do:

1) finish support for C++17 variant header (which is really just fix bugs in the C++17 support)
2) clean up libcxx tests, there are new tests for C++17 and some of the old tests broke.
3) do some final testing for compatibility with cmake (I have a bunch of projects to try to compile)
4) work on static analysis reported bugs (coverity) for the compiler proper.

not sure what the schedule is going to be at this time.    It is taking longer than I would have liked, but, I need to rest sometimes too.

**Comment by LADSoft** [comment]
we are getting closer to this.    After a lot of delays I got the variant header worked out....

**Comment by LADSoft** [comment]
I added some stuff to milestone #4, this is what I want to get completed.   I'm trying to clear out various bugs and technical debt before releasing...

Most of it won't take much effort.

**Comment by LADSoft** [comment]
So we are closing in on a release.   I reviewed the open issues now to see where we are with each of them.   

I think the biggest barrier is probably going to be the libcxx tests, everything else *looks* like it might flow rather smoothly.    Although I expect the CMAKE stuff to take some time as well...

**Comment by LADSoft** [comment]
so the testing is going on and on and on, I'm seriously thinking about having a release of 7.0 sooner rather than later and moving the remaining items in this milestone into another milestone which I will work on right after the release....   then we'll have another release when that is done (maybe I'll make a couple of improvements related to compile speed as well).   About all i want to do at this point is finish up with the covscript compiling...   which I'm down to one file...   but the current bugs are over sixty seconds into the compile when the compiler is run in release mode...   lol....

**Comment by LADSoft** [comment]
the last appveyor build succeeded (other than the libcxx tests) but the last two github actions failed the windows smoke test.   Basically, after the regression tests the runner just hung.   I reran one of those and that fixed it, then I forced another build and that ran through.  Testing on my computer didn't show issues.  We've been battling with randomly failing builds for a very long time and seems like i keep finding valid issues to fix, but it continues to happen.   I'm running out of ideas about that...  

i've enabled debug logging on the github actions so if it happens again maybe I will get enough detail to fix whatever is wrong now...

Meanwhile since it is succeeding now if the appveyor build succeeds (other than the libcxx testing) I will make a release and get back to testing....

**Comment by LADSoft** [comment]
I've been dancing with trying to get the release built.    Between me trying to get the personal access token from github working again and the overall load on appveyor being too high, the builds keep failing...   hopefullly we can get this done soon....

**Comment by chuggafan** [comment]
Well, it looks like v707 has a .exe installer uploaded, so perhaps the latest trick worked?

**Comment by LADSoft** [comment]
yeah it looks like it did work properly this time....    i'll finish the release tomorrow night...
